ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -173,8 +173,13 @@
             class="mt-2"
             hideAddressHint
           />
+          <hr class="mt-3 mb-10" />
+          <HomeOwnerGroups
+            :groupId="ownersGroupId"
+            :isAddingHomeOwner="isAddingHomeOwner"
+            @setOwnerGroupId="updateGroupIdForOwner($event)"
```

Comment:
Pretty minor but if you only had one instance of this method, you could simplify with `ownerGroupId = $event` here

Reply:
Updated.